server/server.py

```diff
@@ -5,10 +5,13 @@
 from fastapi import FastAPI, WebSocket
 import requests
 import json
+from typing import Dict
 
 DJANGO_USERS_URI = 'http://localhost:8000/users'
 DJANGO_GAME_URI = 'http://localhost:8000/games'
 
+EVENT_SEND_CHALLENGE = 'challenge'
```

Comment:
Is it okay to have this here?

Reply:
It will probably go on its own file later, as will the URIs above